.dev/auth/manifests/pod.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: auth
+  labels:
+    app.kubernetes.io/name: auth
+spec:
+  containers:
+    - name: auth
+      image: auth
+      imagePullPolicy: Never # Need this for pushing directly into minikube
+      ports:
+        - containerPort: 9099
+          name: auth-port
+        - containerPort: 9100
+          name: auth-ui-port
+      startupProbe:
+        httpGet:
+          port: auth-ui-port
+          path: /health
+        # Wait 10 sec * 10 fails = 100 seconds before failing
+        failureThreshold: 10
+        periodSeconds: 10
+      resources:
+        limits:
+          cpu: 250m
```

Comment:
What's does `m` mean for CPU?   Ah, "milli".

Reply:
That's correct! 

It's .25 CPU 

https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes